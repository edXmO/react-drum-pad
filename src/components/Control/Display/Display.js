import React from 'react';
import '../../../css/App.css';

const Display = ({ content, screenType }) => {
    return (
        <div className={screenType}>
            {content}
        </div>
    )
}

export default Display;