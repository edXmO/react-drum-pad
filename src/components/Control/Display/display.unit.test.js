import React from 'react';
import renderer from 'react-test-renderer';

import Display from './Display';

describe('should render all buttons without crashing', () => {
    let component = renderer.create(
        <Display type={'VOL: 80%'} screenType={'display display__volume'} />
    );
    it('should render without crashing', () => {
        expect(component.toJSON()).toMatchSnapshot();
    });
});