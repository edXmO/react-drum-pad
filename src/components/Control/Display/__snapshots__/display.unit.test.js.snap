// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render all buttons without crashing should render without crashing 1`] = `
<div
  className="display display__volume"
>
  VOL: 80%
</div>
`;
